core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4352,6 +4360,8 @@ private static String javadocLink(String fullyQualifiedClassname) {
         "alluxio.worker.network.block.reader.threads.max";
     public static final String WORKER_NETWORK_BLOCK_WRITER_THREADS_MAX =
         "alluxio.worker.network.block.writer.threads.max";
+    public static final String WORKER_NETWORK_BLOCK_EVICTION_THREADS =
+        "alluxio.worker.network.block.eviction.threads";
```

Comment:
Does this fit the `network` category?

Reply:
not really. Changed to `WORKER_EVICTOR_BLOCK_TRANSFER_THREAD_COUNT`